examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -434,6 +435,39 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
             // unrecognized map type
             break;
         }
+      },
+
+      querypin: function(pinid){
```

Comment:
The name should be precise. 

pinMapOneTweetQuery: function(tweetid) ... 

Please adjust the related parts, too.

Reply:
ok, will change. Thanks !